app/models/event.rb

```diff
@@ -144,6 +144,14 @@ def date_ranges_attributes=(*args)
     end
   end
 
+  # Returns an semicolon-separated-list of email addresses of all accepted participants.
+  #
+  # @param none
+  # @return [String] for list of email-addresses
+  def get_email_list
+    application_letters.where(status: ApplicationLetter.statuses[:accepted]).reduce('') {|list, letter| list << letter.user.email << ';'}
```

Comment:
Wouldn't this benefit from more line breaks? I wouldn't expect a loop hidden at the end of a line.